src/specs/constants.ts

```diff
@@ -93,6 +94,14 @@ export const DEFAULT_SETTINGS_SPEC: SettingsSpec = {
     type: DEFAULT_TOOLTIP_TYPE,
     snap: DEFAULT_TOOLTIP_SNAP,
   },
+  externalPointerEvents: {
+    tooltip: {
+      visible: false,
+      boundary: 'chart',
```

Comment:
I don't think this should be `chart` by default. I think the default should be the most flexible positioning outside of the chart.

```suggestion
      boundary: undefined, // undefined uses the closest parent scroll container
```